.travis.yml

```diff
@@ -5,5 +5,7 @@ node_js:
   - "4"
   - "5"
   - "6"
+env:
+  - TZ="Europe/Copenhagen"
```

Comment:
To me it would be a good sanity check if the tests run and the assertions produce the same output even if you don't alter the local time zone :)